purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,260 @@
+            ;
+
+            val defaultValue: String
+                get() = when (this) {
+                    NO_THANKS -> "No, thanks"
+                    NO_SUBSCRIPTIONS_FOUND -> "No Subscriptions found"
+                    TRY_CHECK_RESTORE -> "We can try checking your Apple account for any previous purchases"
+                    RESTORE_PURCHASES -> "Restore purchases"
+                    CANCEL -> "Cancel"
+                    BILLING_CYCLE -> "Billing cycle"
+                    CURRENT_PRICE -> "Current price"
+                    EXPIRED -> "Expired"
+                    EXPIRES -> "Expires"
+                    NEXT_BILLING_DATE -> "Next billing date"
+                    REFUND_CANCELED -> "Refund canceled"
+                    REFUND_ERROR_GENERIC -> "An error occurred while processing the refund request. Please try again."
+                    REFUND_GRANTED -> "Refund granted successfully!"
+                    REFUND_STATUS -> "Refund status"
+                    SUB_EARLIEST_EXPIRATION -> "This is your subscription with the earliest expiration date."
+                    SUB_EARLIEST_RENEWAL -> "This is your subscription with the earliest billing date."
+                    SUB_EXPIRED -> "This subscription has expired."
+                    CONTACT_SUPPORT -> "Contact support"
+                    DEFAULT_BODY -> "Please describe your issue or question."
+                    DEFAULT_SUBJECT -> "Support Request"
+                    DISMISS -> "Dismiss"
+                    UPDATE_WARNING_TITLE -> "Update available"
+                    UPDATE_WARNING_DESCRIPTION ->
+                        "Downloading the latest version of the app may help solve the problem."
+                    UPDATE_WARNING_UPDATE -> "Update"
+                    UPDATE_WARNING_IGNORE -> "Continue"
+                    PLATFORM_MISMATCH -> "Platform mismatch"
+                    PLEASE_CONTACT_SUPPORT_TO_MANAGE -> "Please contact support to manage your subscription."
+                    APPLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the App Store app on an Apple device."
+                    GOOGLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the Play Store app on an Android device"
+                    AMAZON_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription in the Amazon Appstore app on an Amazon device."
+                }
+        }
+
+        fun commonLocalizedString(key: CommonLocalizedString): String {
+            return localizedStrings[key.name.lowercase()] ?: key.defaultValue
+        }
+    }
+
+    @Serializable
+    class HelpPath(
+        val id: String,
+        val title: String,
+        val type: PathType,
+        @SerialName("promotional_offer") val promotionalOffer: PathDetail.PromotionalOffer? = null,
+        @SerialName("feedback_survey") val feedbackSurvey: PathDetail.FeedbackSurvey? = null,
```

Comment:
Note that this is a bit different to iOS. In iOS, there is a single `detail` field, which makes sense since you can only have either `promotional_offer` or `feedback_survey`. But that was trickier to deserialize and I believe it's fine to separate these 2 into separate fields. Lmk if you have concerns.

Reply:
For my understanding, in Android we could have both?

Reply:
Not right now since the flow wouldn't make much sense if both are present... Having said that, we're not currently constraining this in the backend, so it's possible to have both.

Reply:
Alright, yea I think these models should reflect the backend as closely as possible, so this is fine with me.